### PR TITLE
Two "bug" fixes related to match()

### PR DESCRIPTION
**Pull request by sandrogauci**
Please see the commit info.

### CODE REVIEW

**Comment by lcmaqueda** [comment]
I think gopass should be consistent with the way pass works. What's the behavior of pass about subdirs starting with dot? Does it index them?

**Comment by sandrogauci** [comment]
@cortex - I agree, it would be more neat to do the truncation on presentation.

@lcmaqueda - pass uses tree in [`cmd_find()`](https://git.zx2c4.com/password-store/tree/src/password-store.sh#n335) which does not search through files or directories starting with a dot. My change only handles directories.

**Comment by lcmaqueda** [comment]
+1 to skipping directories starting with dot. I would even apply the same logic to regular files
+1 to truncation on presentation. Otherwise we should trim the password-store prefix before matching to avoid matching the "/home/username/..."

File: pass.go [hunk 1]
```diff
@@ -83,7 +83,7 @@ func NewPasswordStore() *PasswordStore {
 func (ps *PasswordStore) Query(q string) []Password {
 	var hits []Password
 	for _, p := range ps.passwords {
-		if match(q, p.Name) {
+		if match(q, p.Path) {
```
Comment (lcmaqueda) [inline pass.go:86]: The password-store prefix should be trimmed from the p.Path before matching

**Pull request closed without merging**